components/form/index.js

```diff
@@ -149,7 +175,17 @@ const Form = () => {
               onClick,
             }}
           >
-            Apply To Join
+            {isFormWaiting === false ? (
```

Comment:
Use isFormWaiting in the <Button> component disabled prop as well. Here's an example:

```
<Button disabled={isFormWaiting} onClick={onClick} />
```

Reply:
Updated PR to include this functionality